boards/arm/mimxrt595_evk/doc/index.rst

```diff
@@ -174,6 +174,30 @@ Serial Port
 The MIMXRT595 SoC has 13 FLEXCOMM interfaces for serial communication. One is
 configured as USART for the console and the remaining are not used.
 
+Fusion F1 DSP Core
+==================
+
+You can build a Zephyr application for the RT500 DSP core using nxp_adsp_rt595
+board. Xtensa toolchain and DSP build configuration are required for this.
+For instructions on how to download the toolchain and debug DSP firmware,
+please refer to this document: `Getting Started with Xplorer for EVK-MIMXRT595`_
```

Comment:
This should be updated now that the toolchain has been added to the SDK.

Reply:
updated, thank you!